Manifest.toml

```diff
@@ -0,0 +1,172 @@
+[[Arpack]]
```

Comment:
As I understand it, the Manifest.toml should not be pushed for packages. This would fix all your dependencies to the exact version given here and you would only do this when you want to exactly reproduce someone else's results, etc.

Reply:
good to know, I also added it to .gitignore